embeddings/pipeline/hugging_face_sequence_tagging.py

```diff
@@ -35,5 +35,6 @@ def __init__(
         embedding = FlairTransformerWordEmbedding(embedding_name)
         task = SequenceTagging(output_path, hidden_size=hidden_size)
         model = FlairModel(embedding, task)
-        evaluator = SequenceTaggingEvaluator()
+        # todo: should be parametrised?
+        evaluator = POSTaggingEvaluator()
```

Comment:
@ktagowski, usage of SequenceTaggingEvaluator/POSTaggingEvaluator should be parametrised? maybe a new pipeline class for POS?

Reply:
For SequenceTaggingEvaluator probably there should be parametrization but I will test it with NER pipeline,  I like the idea o new pipeline class for POS

Reply:
So in this PR I would skip parametrization of these metrics and make adapatation on real examples like NER / Political dataset in these task-related PR's.

Reply:
i have already done this separation